File: js/1903-largest-odd-number-in-string.js
/**
 * @param {string} num
 * @return {string}
 */
var largestOddNumber = function (num) {
  for (var i = num.length - 1; i >= 0; i--) {
    if (num[i] % 2 === 1) return num.substring(0, i + 1);
  }
  return "";
};

module.exports = { largestOddNumber };

File: go/solutions/1903-largest-odd-number-in-string.go
package solutions

func LargestOddNumber(num string) string {
	for i := len(num) - 1; i >= 0; i-- {
		if int(num[i]-'0')%2 == 1 {
			return string(num[:i+1])
		}
	}

	return ""
}
